src/Infrastructure/KafkaJanitor.RestApi/Features/Vault/AwsSsmParameterStoreVault.cs

```diff
@@ -17,7 +17,7 @@ public async Task AddApiCredentials(Capability capability, ApiCredentials apiCre
             await ssmClient.PutParameterAsync(new PutParameterRequest
             {
                 Type = ParameterType.SecureString,
-                Name = $"/capabilities/{capability.Id}/kafka/credentials",
+                Name = $"/capabilities/{capability.RootId}/kafka/credentials",
```

Comment:
Do we need to transform rood id to ensure it is safe for aws?

Reply:
Nope. It's safe.